.isort.cfg

```diff
@@ -4,6 +4,6 @@ indent='    '
 multi_line_output=4
 default_section=FIRSTPARTY
 known_firstparty=readthedocs,readthedocsinc
-known_third_party=celery,stripe,requests,pytz,builtins,django,annoying,readthedocs_build
+known_third_party=celery,stripe,requests,pytz,builtins,django,annoying,mock,allauth,oauthlib,requests_oauthlib,readthedocs_build
```

Comment:
Just curious -- why do we need to maintain this? Shouldn't it know what is third party based on the package not being first party? I'd like to avoid maintaining a list here

Reply:
This should be intelligent enough and we shouldn't update this list with all the third party packages (I know that sometimes it gets confused because `mock` for example is a third party package in py2 but an stdlib in py3)

I'm not sure why it's not working as I expected. I need to research a little more about this. I'm using https://github.com/FalconSocial/pre-commit-python-sorter and I'm not sure that it's taking the configuration in a proper way.

Reply:
Aha! I think it's because 

    default_section=FIRSTPARTY

and maybe it should be THIRDPARTY there :)

> default_section: The default section to place imports in, if their section can not be automatically determined.

https://github.com/timothycrosley/isort/wiki/isort-Settings

Reply:
Should be fixed now.